<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic_fitting_widget</class>
 <widget class="QWidget" name="basic_fitting_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="fit_button">
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="undo_fit_button">
        <property name="text">
         <string>Undo Fits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fit_generator_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Fit Generator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="global_fit_status_label">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string>No Fit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="plot_guess_checkbox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Plot guess</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
